server/vulnerabilities/nvd/cpe_test.go

```diff
@@ -466,8 +466,6 @@ func TestLegacyCPEDB(t *testing.T) {
 }
 
 func TestCPEFromSoftwareIntegration(t *testing.T) {
-	nettest.Run(t)
```

Comment:
did you remove this for local testing?

Reply:
I actually moved it below, and is called if `NVD_TEST_CPEDB_PATH` is not set.